sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -466,7 +467,7 @@ For more information see the [Code of Conduct FAQ][coc_faq] or contact opencode@
 
 <!-- [secrets_client_library]: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/keyvault/Azure.Security.KeyVault.Secrets -->
 
-[soft_delete]: https://docs.microsoft.com/azure/key-vault/key-vault-ovw-soft-delete
+[soft_delete]: https://docs.microsoft.com/azure/key-vault/general/soft-delete-overview
```

Comment:
Similar change was made in other repos:
https://github.com/Azure/azure-sdk-for-net/pull/22772

@heaths

I think python needs to be fixed too (FYI  @schaabs @chlowell @mccoyp):
https://github.com/Azure/azure-sdk-for-python/blob/264cd5d1aec86cf7b550f141fa93e3f224dfc7e6/sdk/keyvault/azure-keyvault-secrets/README.md#delete-a-secret